packages/webpack/src/utils/module-federation/start-remote-proxies.ts

```diff
@@ -1,25 +1,46 @@
 import type { Express } from 'express';
-import { logger } from '@nx/devkit';
+import { logger, ProjectGraph } from '@nx/devkit';
```

Comment:
Do you need the `ProjectGraph` ref here?

Reply:
Nope, fixed, good catch